auditbeat/Makefile

```diff
@@ -11,3 +11,7 @@ include ${ES_BEATS}/libbeat/scripts/Makefile
 .PHONY: update
 update: mage
 	mage update
+
+.PHONY: unit
+unit: mage
+	mage unitTest
```

Comment:
Do we want/need the make targets?

I guess the only reason to have them is to keep `make unit` working from the root directory, wdyt about replacing the call to `make unit` there with a call to the `mage` target for each project? 

If we want to have the same command in the root directory and in each project, instead of adding make targets to all projects, we could add a `mage unitTest` target to the root `magefile.go`, that calls this target on the individual projects.

Reply:
Agreed. Let's look for consistency in the _official_ build system

Reply:
@jsoriano I updated the PR with the new `mage` goal, reverting the `make` goals. Please let me now what do you think, thanks!

Reply:
Thanks!